OpenSim/Simulation/Model/ModelVisualizer.h

```diff
@@ -194,6 +197,11 @@ class OSIMSIMULATION_API ModelVisualizer {
                             const std::string&          geoFile,
                             bool&                       isAbsolute,
                             SimTK::Array_<std::string>& attempts);
```

Comment:
It might be nice to add to the doxygen description that the last entry of `attempts` is the correct path of the file.